app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
If you don't put it in there, hound complains. Apparently it does speed up performance by making strings immutable.